tedium.js

```diff
@@ -27,7 +29,41 @@ const path = require('path');
 const ProgressBar = require('progress');
 const rimraf = require('rimraf');
 const cleanup = require('./cleanup-passes');
-
+const hydrolysis = require('hydrolysis');
+const pad = require('pad');
+const cliArgs = require("command-line-args");
+const cli = cliArgs([
+  {
+    name: "help",
+    type: Boolean,
+    alias: "h",
+    description: "Print usage."
+  },
+  {
+    name: "max_changes",
```

Comment:
It might be nice to have something like a `--dry-run` option where it does everything except push out the final changes. Or something to that effect.

But otherwise, thanks for the explanation. SGTM.